PhotoKeeper/PhotoViewController.swift

```diff
@@ -9,37 +9,73 @@
 import UIKit
 
 class PhotoViewController: UIViewController {
-    var image: UIImage?
-    var imageTitle: String?
-    var imageDescription: String?
+
+    let store = PhotoStore.shared
+    var photo: PhotoObject!
```

Comment:
photo should probably be `weak`, optional, and have a `didSet`

```swift
weak var photo: PhotoObject? {
  didSet {
    title = photo.title
    // set image in image view
    buildMetaDataView()
  }
}
```

you'll have to make `iv` below a class property, but otherwise you shouldn't have to do too much different.

This will make your view controller more flexible and less error prone.

Reply:
this pattern confuses me becuase now its making me force unwrap my photo object inside of didSet ...
![image](https://user-images.githubusercontent.com/7544345/41003250-48432d60-68e4-11e8-8ad7-a5d9a38cc5d6.png)